bin/node/cli/tests/common.rs

```diff
@@ -65,3 +65,39 @@ pub fn run_dev_node_for_a_while(base_path: &Path) {
 	kill(Pid::from_raw(cmd.id().try_into().unwrap()), SIGINT).unwrap();
 	assert!(wait_for(&mut cmd, 40).map(|x| x.success()).unwrap_or_default());
 }
+
+/// Run the node for a while (30 seconds)
+pub fn run_node_with_args_for_a_while(base_path: &Path, args: &[&str]) {
```

Comment:
`run_dev_node_for_a_while` could be implemented in terms of `run_node_with_args_for_a_while` now

Reply:
eg https://github.com/cheme/substrate/blob/844fb80e0195ed05a2ecc1d95baa0c5760278388/bin/node/cli/tests/common.rs#L50